SingleFrac_EFP_vs_LIE/single_fracture_EFP.prj

```diff
@@ -0,0 +1,349 @@
+                <!-- <compensate_non_equilibrium_initial_residuum>true</compensate_non_equilibrium_initial_residuum> -->
+                <time_discretization>
+                    <type>BackwardEuler</type>
+                </time_discretization>
+                <time_stepping>
+                    <type>FixedTimeStepping</type>
+                    <t_initial>0</t_initial>
+                    <t_end>2000</t_end>
+                    <timesteps>
+                        <pair>
+                            <repeat>20</repeat>
+                            <delta_t>100</delta_t>
+                        </pair>
+                    </timesteps>
+                </time_stepping>
+            </process>
+        </processes>
+        <output>
+            <type>VTK</type>
+            <prefix>single_fracture_EFP</prefix>
+            <timesteps>
+                <pair>
+                    <repeat>1</repeat>
+                    <each_steps>1</each_steps>
+                </pair>
+            </timesteps>
+            <variables>
+                <variable>pressure</variable>
+                <variable>displacement</variable>
+            </variables>
+        </output>
+    </time_loop>
+    <media>
+        <medium id="0">
+            <phases>
+                <phase>
+                    <type>Gas</type>
+                    <properties>
+                        <property>
+                            <name>viscosity</name>
+                            <type>Constant</type>
+                            <value>0.001</value>
+                        </property>
+                        <property>
+                            <name>density</name>
+                            <type>Linear</type>
+                            <independent_variable>
+                                <variable_name>phase_pressure</variable_name>
+                                <reference_condition>0</reference_condition>
+                                <slope>1e-07</slope>
+                            </independent_variable>
+                            <reference_value>1000.0</reference_value>
+                        </property>
+                    </properties>
+                </phase>
+                <phase>
+                    <type>Solid</type>
+                    <properties>
+                        <property>
+                            <name>density</name>
+                            <type>Constant</type>
+                            <value>2716.0</value>
+                        </property>
+                        <property>
+                            <name>porosity</name>
+                            <type>Constant</type>
+                            <value>0.001</value>
+                        </property>
+                        <property>
+                            <name>biot_coefficient</name>
+                            <type>Constant</type>
+                            <value>1</value>
+                        </property>
```

Comment:
These are now medium properties, no longer solid phase. Please move to medium for running with master